dashboard/src/components/ApiDocs/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Auth } from "shared/Auth";
+import SwaggerUI from "swagger-ui-react";
+import "swagger-ui-react/swagger-ui.css";
+
+// Request interface needed for avoiding type error in the requestInterceptor
+// it is being used, but it is not exported, so we define it here
+// tslint:disable-next-line
+interface Request {
+  [k: string]: any;
+}
+
+function authenticate(req: Request): Request {
+  const token = Auth.getAuthToken() || "";
+  // Otherwise, OIDC login is being used and we cannot automatically set the token here
```

Comment:
If the user is logged in, the cookie will be set (you can check) and the auth-proxy will be updating the request with the Authorization header anyway (ie. the user can test the API as themselves). Happy to check that once landed.